### DISCUSSION TITLE
hideChat 처리 리밋이 있나요?

### DISCUSSION THREAD

**Discussion by mschadev** [storycraft/node-kakao#358] [Q&A] [answered]
두 가지 질문이 있습니다.
1. 다량의 메시지를 연속으로 hideChat하면 요청 res도 모두 정상인데 실제로는 일부 메시지가 가려지지 않은 상태입니다.  
  리밋이 있는지 궁금합니다.
2. 이미 가려진 메시지에 대해 다시 한번 hideChat을 하면 요청이 정상으로 뜨는데, 정상인지 궁금합니다.

버전은 `4.2.2`고, 항상 잘 사용하고 있습니다. 감사합니다.

**Comment by storycraft** [comment, marked as answer]
1. 카카오톡쪽 버그로 보입니다. 실제 클라이언트에서도 채팅 가리기 관련 패킷을 정상적으로 다 받는 것으로 확인했습니다.
2. 정상입니다.

**Comment by JellyBrick** [comment]
1. 약간의 딜레이를 주고 가려야 하는 것으로 보입니다.

**Reply by mschadev** [reply]
추가 답변 감사합니다.

**Reply by MoonJR** [reply]
현재 저는 0.5초간격으로 딜레이를 주어 3번 동작하도록 수행하니 그나마 좀 괜찮네요
